ui/apps/portal/src/plugins/SystemAdmin/StudyPage/StudyPage.tsx

```diff
@@ -0,0 +1,202 @@
+import { Loader, MenuItem, Select, SelectChangeEvent } from "@portal/components";
+import { PageProps, SystemAdminPageMetadata } from "@portal/plugin";
+import React, { FC, useCallback, useEffect, useState } from "react";
+import { api } from "../../../axios/api";
+import { useFeedback, useTranslation } from "../../../contexts";
+import { useDatasets } from "../../../hooks";
+import { StrategusStudy } from "../../../types";
+import { StudyCard } from "./StudyCard";
+import "./StudyPage.scss";
+
+interface StudyPageProps extends PageProps<SystemAdminPageMetadata> {}
+
+interface StrategusStudiesData {
+  [studyId: string]: StrategusStudy;
+}
+
+export const StudyPage: FC<StudyPageProps> = () => {
+  // TODO: Add translation
+  const { getText, i18nKeys } = useTranslation();
+  const { setFeedback } = useFeedback();
+  const [datasets, loadingDatasets, error] = useDatasets("systemAdmin");
+  const [selectedDatasetId, setSelectedDatasetId] = useState<string>("");
+  const [strategusStudies, setStrategusStudies] = useState<StrategusStudy[]>([]);
+  const [loadingStudies, setLoadingStudies] = useState<boolean>(false);
+  const [studiesError, setStudiesError] = useState<string | null>(null);
+  const [runningStudyId, setRunningStudyId] = useState<string | null>(null);
+
+  const handleDatasetChange = useCallback((event: SelectChangeEvent) => {
+    setSelectedDatasetId(event.target.value);
+  }, []);
+
+  useEffect(() => {
+    if (!selectedDatasetId) {
+      setStrategusStudies([]);
+      return;
+    }
+
+    const fetchStudies = async () => {
+      setLoadingStudies(true);
+      setStudiesError(null);
+
+      try {
+        const studiesData = (await api.systemPortal.getStudiesFromRepo()) as StrategusStudiesData;
+        const convertedStudies: StrategusStudy[] = Object.entries(studiesData).map(
+          ([studyId, strategusStudy]: [string, StrategusStudy]) => ({
+            ...strategusStudy,
+            id: studyId,
+            name: strategusStudy.name || studyId,
+          })
+        );
+
+        setStrategusStudies(convertedStudies);
+      } catch (error) {
+        console.error("Error fetching studies from repository:", error);
+        setStudiesError("Failed to fetch studies from repository");
+        setStrategusStudies([]);
+      } finally {
+        setLoadingStudies(false);
+      }
+    };
+
+    fetchStudies();
+  }, [selectedDatasetId, setFeedback]);
+
+  const handleRunStudy = useCallback(
+    async (study: StrategusStudy) => {
+      if (runningStudyId) {
```

Comment:
Do we need to handle running multiple studies?

Reply:
It would make sense to move logic for running strategus (including view result, share result etc) to inside the study card so that we can manage the study run individually